src/app/components/grid/client.ts

```diff
@@ -0,0 +1,126 @@
+import { Socket, Channel } from 'phoenix';
+import { Growls } from '../../../lib/gj-lib-client/components/growls/growls.service';
+import { Environment } from '../../../lib/gj-lib-client/components/environment/environment.service';
+import * as nwGui from 'nw.gui';
+import { store } from '../../store/index';
+import {
+	Notification,
+	getNotificationText,
+} from '../../../lib/gj-lib-client/components/notification/notification-model';
+
+function getCookie(name: string): Promise<string | undefined> {
```

Comment:
what do you think about putting this into a new file and exporting it? maybe in frontend-lib/components/cookie/cookie.service.ts or something? it can just export this one func, but it would be nice since we use it both here and in chat now

Reply:
actually, maybe not frontend-lib since it depends on nw.gui which is included only in gamejolt repo

Reply:
i did this